core/src/main/resources/hudson/model/Run/console-log.jelly

```diff
@@ -0,0 +1,38 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:t="/lib/hudson">
+  <j:set var="threshold" value="${h.getSystemProperty('hudson.consoleTailKB')?:'150'}" />
+  <!-- Show at most last 150KB (can override with system property) unless consoleFull is set -->
+  <j:set var="offset" value="${empty(consoleFull) ? it.logText.length()-threshold*1024 : 0}" />
+  <j:choose>
+    <j:when test="${offset > 0}">
+      <a class="jenkins-button jenkins-!-accent-color jenkins-!-padding-2 jenkins-!-margin-bottom-2" style="width: 100%; justify-content: start" href="consoleFull">
+        <l:icon src="symbol-help-circle" />
```

Comment:
under a UX point of view I am not sure whether the `?` makes any sense I feel `<l:icon src="symbol-information-circle" />` makes more sense in the context